mlir/include/mlir/Conversion/Passes.td

```diff
@@ -1394,6 +1394,11 @@ def ConvertVectorToLLVMPass : Pass<"convert-vector-to-llvm"> {
            "bool", /*default=*/"true",
            "Allows compiler to assume vector indices fit in 32-bit if that "
      "yields faster code">,
+    Option<"useVectorAlignment", "use-vector-alignment",
```

Comment:
Given that we don't test any hardware upstream that would require this, we should make sure that the rationale for this flag is clearly documented.

Could you extend this with a note saying that this is for hardware which requires "vector alignment"?

Reply:
I added some more explanation!